tests/software/lofreq/callparallel/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: lofreq callparallel
+  command: nextflow run ./tests/software/lofreq/callparallel -entry test_lofreq_callparallel -c tests/config/nextflow.config
+  tags:
+    - lofreq
+    - lofreq/callparallel
+  files:
+    - path: output/lofreq/test.vcf
+      md5sum: 5bfa1c927ab131c6dc1109bb41dc5992
```

Comment:
```suggestion
```

Reply:
Ok. So it appears that there is something variable that is added to the VCF file which is causing the md5sum checks to fail.